Makefile

```diff
@@ -61,7 +61,7 @@ IMAGE = $(REGISTRY)/$(IMGNAME)
 MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
 
 # Set default base image dynamically for each arch
-BASEIMAGE?=quay.io/kubernetes-ingress-controller/nginx-$(ARCH):0.73
+BASEIMAGE?=quay.io/kubernetes-ingress-controller/nginx-$(ARCH):0.74
```

Comment:
I don't see this version of Nginx image at https://github.com/kubernetes/ingress-nginx/blob/1e2146c26a44ae33a3dc8902e8db57bee0a8ead7/images/nginx/Makefile#L16, am I missing something?